bootstrap/src/constants.js

```diff
@@ -3,4 +3,5 @@ module.exports = {
     appName: "My App",
     backend: "https://developer.na.gooddata.com",
     projectId: "",
+    projectFilter: "", // regexp to filter projects in ProjectPicker. Example: /test/ or /^GoodData/,
```

Comment:
In the comment we say it  should be a regex, yet we default to a string. It seems inconsistent, can we dafault to either a RegExp or null?